.github/main.workflow

```diff
@@ -0,0 +1,18 @@
+workflow "on push" {
+  on = "push"
+  resolves = ["Find Python 3 syntax errors and undefined names", "GitHub Action for Flake8", "GitHub Action for pylint"]
```

Comment:
Let’s comment out the failing test and add one that should pass...

Please comment out line 3 for now and add:
```
  resolves = ["GitHub Action for cpplint"]
}

action "GitHub Action for cpplint" {
  uses = "cclauss/GitHub-Action-for-cpplint@master"
  args = "cpplint --recursive ."
}
```